shared/js/initApp.js

```diff
@@ -1,9 +1,21 @@
+// TODO (brent) - way too many globals
+// TODO (brent) - I wonder if we should sub-namespace dashboard
+/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Blockly, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga*/
```

Comment:
I'd love to break this file up further.  It initially was just the JS from the `_blockly.html.haml` view [[1](https://github.com/code-dot-org/code-dot-org/commit/cac99fae7a0e1baf46fd27f635d0607006e394c6)] [[2](https://github.com/code-dot-org/code-dot-org/commit/58343c94e93f07d53e99aa04d71f06f7944801fa)] but it's grown as a catch-all.

The best candidate is probably the project logic, which can go into a different file.  I'll take a look.